### PR TITLE
Cyberprodigy hometown

### LINKED ISSUE #30: Create cyberprodigy hometown file
## Gaujiena, LV
- [x] Create a branch
- [x] Create a file
- [x] Add a commit
- [x] Create a Pull Request
- [x] Have a conveersation
- [x] Get a :+1: 
- [x] :ship: it (a.k.a. merge)

### PR DESCRIPTION
**Pull request by cyberprodigy**
This PR resolves #30 
~Pandalove

### CODE REVIEW

File: cyberprodigy-gaujiena.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Gaujiena, LV
+
+### Places to eat
+- Melderitis
+Party rooms (60 persons), meals, party service, caf�, volleyball grounds, boating down the Gauja river.
+
+
+### Fun things to do
+- Play some sports
```
Comment (cyberprodigy) [inline cyberprodigy-gaujiena.md]: I can't wait to hear what you recommend!

**Comment by phanimorampudi** [comment]
commented

**Comment by phani-morampudi** [comment]
tretr

File: cyberprodigy-gaujiena.md [hunk 2]
```diff
@@ -6,9 +6,4 @@ Party rooms (60 persons), meals, party service, caf
 
 
 ### Fun things to do
-- Play some sports
-
-
-<script>
-alert("");
-</script>
```
Comment (antonbuijs) [inline cyberprodigy-gaujiena.md]: OK

File: cyberprodigy-gaujiena.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+## Gaujiena, LV
+
+### Places to eat
+- Melderitis
```
Comment (antonbuijs) [inline cyberprodigy-gaujiena.md:4]: OK

**Comment by szamanx** [comment]
:+1:

**Pull request merged**